templates.dist/fields.php

```diff
@@ -133,7 +134,7 @@ function renderFormField(string $fieldName, onOffice\WPlugin\Form $pForm, bool $
 		if (\onOffice\WPlugin\Types\FieldTypes::FIELD_TYPE_SINGLESELECT == $typeCurrentInput) {
 			$output .= '<select class="custom-single-select" size="1" name="' . esc_html($fieldName) . '" ' . $requiredAttribute . '>';
 			/* translators: %s will be replaced with the translated field name. */
-			$output .= '<option value="">' . esc_html(sprintf(__('Choose %s', 'onoffice'), $fieldLabel)) . '</option>';
+			$output .= '<option value="">' . esc_html(sprintf(__('Choose %s', 'onoffice'), $fieldDefault)) . '</option>';
```

Comment:
This line already covered the translation

Reply:
So `$fieldDefault` is translated? Then it's all good. :)

Reply:
Yes

Reply:
Yes! And I think the problem here is that the translations for the labels should not come from WordPress but from onOffice enterprise.

I think #266 was a similar situation, so maybe you can find information there.

Reply:
API doesn't return the default field. that why I need to create a default array for the name of option default.

Reply:
Do you have any update on this issue.

Reply:
Thanks for the reminder, I forgot to answer!

What is this default then? What do you use it for?

Reply:
The default is the un-change label likes "GDPR status". API doesn't return the original label (default), so I make a hard code default fields to list all original labels.

Reply:
I think I'm not understanding what you mean. I can change the label in onOffice enterprise and it will change on the frontend as well.

![grafik](https://user-images.githubusercontent.com/90763364/178427735-ff461e50-c3e3-423b-8f3d-eca74081b7da.png)

Let's clear this up in the weekly on Wednesday. ;)

Reply:
This is the bug you are mention. We should keep the default label for the default option of a select box. But in the API response I can't find some information about "GDPR status", that why I need to make a hardcode list (for the default option name).

Reply:
![image](https://user-images.githubusercontent.com/106214469/178435375-faf56be5-3560-4539-9355-b32d7a543011.png)
This is your expectation. Please correct me if I'm wrong. Do we need to keep the default option of the select as the original label or it is just a mistake from your expectation so we need apply the change for the default option also?

Reply:
As discussed in the weekly, this is a mistake in my requirements. Both the label and the dropdown should show the same label.